src/Register/index.test.js

```diff
@@ -1,91 +1,148 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { BrowserRouter as Router } from 'react-router-dom';
+import { MemoryRouter as Router, Switch, Route } from 'react-router-dom';
 import { MockedProvider } from '@apollo/client/testing';
 import AuthProvider from '../AuthProvider';
 import Register from './index';
 import QUERY_USERS from '../graphql/queries';
 import REGISTER from '../graphql/mutations';
+import { HOME_PATH, REGISTER_PATH } from '../paths';
 
-const mockedUsersResponse = {
-  request: { query: QUERY_USERS },
-  result: {
-    data: {
-      users: [
-        {
-          username: 'User',
-          password: 'password',
-          id: 1,
-        },
-      ],
+const mocks = [
+  {
+    request: { query: QUERY_USERS },
+    result: {
+      data: {
+        users: [
```

Comment:
Same as above, it would be great if we can have multiple users in here or achieve that in a different way!